ROMFS/px4fmu_common/init.d-posix/px4-rc.jmavsim

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
+# shellcheck disable=SC2154
```

Comment:
Why?

Reply:
CI fails without it, carried over from here
https://github.com/PX4/PX4-Autopilot/pull/24600/files#diff-0d4c77fd4a6b4056f7689aa9e02d5d52525f4a0e6108bc32cbd8b8b880bcccfdL2